test/server/cards/06_SEC/events/DismantleTheConspiracy.spec.ts

```diff
@@ -0,0 +1,54 @@
+describe('Dismantle the Conspiracy', function() {
+    integration(function(contextRef) {
+        it('Dismantle the Conspiracy\'s ability should allow you to capture any number of enemy non-leader units with 7 or less HP', async function () {
+            await contextRef.setupTestAsync({
+                phase: 'action',
+                player1: {
+                    hand: ['dismantle-the-conspiracy'],
+                    groundArena: [{ card: 'battlefield-marine', damage: 2 }, 'isb-agent']
+                },
+                player2: {
+                    hand: ['change-of-heart'],
+                    groundArena: ['wampa', 'greedo#slow-on-the-draw', 'death-star-stormtrooper'],
+                    spaceArena: ['wing-leader', 'consortium-starviper', 'avenger#hunting-star-destroyer'],
+                    leader: { card: 'iden-versio#inferno-squad-commander', deployed: true },
+                }
```

Comment:
can we start with Player 1 having a deployed leader, so we can ensure the leader unit is a valid target for the `friendlyUnit` resolver?